landscape/categories.md

```diff
@@ -18,6 +20,8 @@ sub-categories. The remaining text is a description of each sub-category.
   - _Pipelines_
     - Tools that ensure a secure pipeline or workflow, for example,
     as applied to devops (CI/CD), supply chain, etc.
+  - _Secret Management_
+    - Tools that help to manage, encrypt or distribute secrets and other sensitive information.
```

Comment:
I think this is the same as "Key and Certificate Management" (which I think is in the other section since it is needed across all parts of cloud technology, not just "App definition and development")

Reply:
It's not necessarily the same I think, but maybe it can both go into more generic categories. Keys and certificates (at least IMO) are more areas of TLS/secure communication. Maybe secret distribution system? This is the name used by Istio I think.

Reply:
Would be helpful to have some examples of open source projects that fall into this category vs Key and Certificate Management

Reply:
There are a few, projects like [Hashicorp Vault](https://www.vaultproject.io/), [Sealed Secrets](https://github.com/bitnami-labs/sealed-secrets) or [Kamus](https://kamus.soluto.io) (which is also proposed as a CNCF project). Vault can do also key management IIRC, and there is also [Lemur](https://github.com/Netflix/lemur).